api/webhook/verify.go

```diff
@@ -0,0 +1,49 @@
+package webhook
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/hex"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+func verifyAndGetPayload(r *http.Request) ([]byte, error) {
```

Comment:
Optional: this is fine for now, but I imagine you'd like to reuse this method in checks to verify the payload? (I saw your TODO in that PR.)

When that happens, we'd need to export (and perhaps move) this function, and add `tokenName` as an argument.

Reply:
Will do :)